src/main/java/com/wavefront/sdk/common/clients/WavefrontClient.java

```diff
@@ -123,6 +123,16 @@ public Builder(String server, @Nullable String token) {
       this.token = token;
     }
 
+    /**
+     * Create a new WavefrontClient.Builder
+     *
+     * @param proxyServer A server URL of the the form "http://internal.proxy.com:port"
```

Comment:
the example should be proxy://internal.proxy.com:port to make it clearer (we understand the http to proxy concept well but probably not all customers will be)

Reply:
@panghy the problem with proxy:// at this point is that the factory figures this out for us and handles the builder logic.  We can accept the proxy:// here but that logic is not at this level.

Reply:
as in we'll have to pull it down into the builder from the factory, since the goal was to abstract those oddities from the builder.

Reply:
okay, let's leave it on http then.